src/anchorediting.js

```diff
@@ -100,15 +100,20 @@ export default class AnchorEditing extends Plugin {
 			}
 		});
 
-		editor.conversion.for( 'editingDowncast' )
-			.attributeToElement( { model: 'anchorId', view: ( id, conversionApi ) => {
-				if (id) {
-					return createAnchorElement( ensureSafeUrl( id ), conversionApi );
-				}
-				else {
+		editor.conversion.for('editingDowncast').attributeToElement({
+			model: 'anchorId',
+			view: (id, conversionApi) => {
+				if (!id) {
 					return null;
 				}
-			} } );
+				const { writer } = conversionApi;
+				const safeId = ensureSafeUrl(id);
+				const anchorElement = writer.createAttributeElement('a', { id: safeId }, { priority: 5 });
```

Comment:
Instead of duplicating, will it work if it still calls `createAnchorElement` and then adds the class to what is returned from that?

E.g something like this (not tested, pseudo code)

```
const { writer } = conversionApi;
const anchorElement = createAnchorElement( ensureSafeUrl( id ), conversionApi );
writer.addClass("ck-anchor", anchorElement);
return anchorElement
```